docs/clusterdefinition.md

```diff
@@ -68,6 +68,9 @@ To learn more about supported orchestrators and versions, run the orchestrators
 | serviceCidr                     | no       | IP range for Service IPs, Default is "10.0.0.0/16". This range is never routed outside of a node so does not need to lie within clusterSubnet or the VNET                                                                                                                                                                                                                                                     |
 | useInstanceMetadata             | no       | Use the Azure cloudprovider instance metadata service for appropriate resource discovery operations. Default is `true`                                                                                                                                                                                                                                                                                        |
 | useManagedIdentity              | no       | Includes and uses MSI identities for all interactions with the Azure Resource Manager (ARM) API. Instead of using a static service principal written to /etc/kubernetes/azure.json, Kubernetes will use a dynamic, time-limited token fetched from the MSI extension running on master and agent nodes. This support is currently alpha and requires Kubernetes v1.9.1 or newer. (boolean - default == false). When MasterProfile is using `VirtualMachineScaleSets`, this feature requires Kubernetes v1.12 or newer as we default to using user assigned identity. |
+| azureCNIVersion                 | no       | Deploy a specific Azure CNI version instead of the default. Example: "v1.0.14" |
```

Comment:
This is a read-only property currently (populated during template generation), it doesn't actually operate as a user toggle to install a specific version of Azure CNI.

Reply:
aha - I see that now. No error is returned but it didn't work